raiden/tests/integration/api/test_restapi.py

```diff
@@ -1291,3 +1291,51 @@ def test_payment_events_endpoints(api_backend, raiden_network, token_addresses):
     response = response.json()
     assert len(response) == 1
     assert response[0]['event'] == 'EventPaymentSentSuccess'
+
+
+@pytest.mark.parametrize('number_of_nodes', [2])
+def test_channel_transfer_events(api_backend, raiden_network, token_addresses):
+    _, app1 = raiden_network
+    amount = 200
+    identifier = 42
+    token_address = token_addresses[0]
+
+    target1_address = app1.raiden.address
+
+    api_server, _ = api_backend
+
+    # payment 1
+    request = grequests.post(
+        api_url_for(
+            api_backend,
+            'token_target_paymentresource',
+            token_address=to_checksum_address(token_address),
+            target_address=to_checksum_address(target1_address),
+        ),
+        json={'amount': amount, 'identifier': identifier},
+    )
+    request.send()
+
+    # payment 2
+    request = grequests.post(
+        api_url_for(
+            api_backend,
+            'token_target_paymentresource',
+            token_address=to_checksum_address(token_address),
+            target_address=to_checksum_address(target1_address),
+        ),
+        json={'amount': amount, 'identifier': identifier},
+    )
+    request.send()
+
+    request = grequests.get(
+        api_url_for(
+            api_backend,
+            'tokeneventsresource',
+            token_address=token_address,
+            from_block=0,
+        ),
+    )
+    response = request.send().response
+    assert_proper_response(response, status_code=HTTPStatus.OK)
+    assert len(response.json()) > 0
```

Comment:
Can you check that the expected events are there and contain the expected fields

Reply:
Ofc